### PR TITLE
Little optimisations in ibxm

### PR DESCRIPTION
**Pull request by Valchap**
Everything is in the title

### CODE REVIEW

File: src/main/java/ibxm/Sample.java [hunk 1]
```diff
@@ -229,11 +229,10 @@ public void resample_sinc(
 	}
 
 	public boolean has_finished( int sample_idx ) {
-		boolean finished;
-		finished = false;
 		if( loop_length <= 1 && sample_idx > loop_start ) {
-			finished = true;
+			return true;
+		} else {
+			return false;
```
Comment (liach) [inline src/main/java/ibxm/Sample.java]: This shows you don't know about optimization at all.

Comment (Valchap) [reply src/main/java/ibxm/Sample.java]: Why is it slower ?

Comment (liach) [reply src/main/java/ibxm/Sample.java]: Don't you know that you can just return the thing in if parentheses?

Comment (Valchap) [reply src/main/java/ibxm/Sample.java]: You are right. I did not think about it

**Comment by LexManos** [comment]
1) Some of these 'optimizations' literally compile to the same bytecode. Or are so small that it does not matter.
2) ixbm is a in-source library, we are not modifying it as to keep it as close to the library we are using as possible.

**Pull request closed without merging**